src/distribution/geometric.rs

```diff
@@ -68,17 +68,23 @@ impl Geometric {
     }
 }
 
```

Comment:
Would you be open to a remark in the docstring mentioning how this uses f64 internally but that accurate values would be within u64? Perhaps something similar for hypergeometric if that's a feasible analysis.

Reply:
I’ve added a comment to explain it. I don’t think it applies to hypergeometric because the implementation of that is just adding 1 to an integer a number of times, which will never overflow a `u64` anyway.